src/app/components/Form/AssetSelect/index.tsx

```diff
@@ -26,10 +26,8 @@ export function AssetSelect(props: Props) {
   return (
     <Select
       value={props.value as any}
```

Comment:
Select should accept a nullish value, no casting should be required.

Reply:
I wanted to change it but unfortunately, we don't have any current usage of this. However, I can change it and we can adjust the usage later.

Reply:
Okey.

Reply:
Changed + `Select` was adjusted as well.